submissions/busolaAdedoyin/css/style2.css

```diff
@@ -0,0 +1,229 @@
+ body {
```

Comment:
well done. Rename this file from `style2.css` to `contact.css` instead

Reply:
@adaugochi I have renamed the file to `contact.css`